src/flows/pipes/Notification/PagerDuty.tsx

```diff
@@ -43,7 +43,7 @@ const PagerDuty: FC = () => {
       <Form.Element label="Routing Key">
         <Input
           name="key"
-          type={InputType.Text}
+          type={InputType.Password}
```

Comment:
changed this for consistency. Not married to the idea, but we should consider allowing the user to toggle visibility here

Reply:
there's a greater arc where we're supposed to have support for secrets in here, but that hasn't come from design yet, so this is totally fine